src/styles/globals.css

```diff
@@ -65,11 +38,6 @@ body {
     background-color: rgb(238, 236, 251);
 }
 
-.footer-pos {
-    display: flex;
-    justify-content: center;
-}
-
 .body {
```

Comment:
This could also be migrated to Tailwind classes wherever the `<body>` element gets rendered. Also it may be that the only actually necessary prop here is `min-height`.